File: apps/server/internal/application/dto/response/owner.go
package response

import (
	"imobiliary/internal/domain/types"
)

type OwnerDTO struct {
	ID            string              `json:"id"`
	ManagerID     string              `json:"manager_id"`
	Fullname      string              `json:"fullname"`
	CPF           string              `json:"cpf"`
	RG            string              `json:"rg"`
	Email         string              `json:"email"`
	Phone         string              `json:"phone"`
	Occupation    string              `json:"occupation"`
	MaritalStatus types.MaritalStatus `json:"marital_status"`
	Address       AddressDTO          `json:"address"`
}

File: apps/web/types/owner.ts
export type Owner = {
  id: string
  manager_id: string
  fullname: string
  cpf: string
  rg: string
  email: string
  phone: string
  occupation: string
  marital_status: 'Solteiro(a)' | 'Casado(a)' | 'Amasiado(a)' | 'Divorciado(a)' | 'Viúvo(a)' | 'União Estável',
  address: {
    address: string
    street: string
    number: string
    neighborhood: string
    complement: string
    city: string
    state: string
    state_abbr: string
    zip_code: string
  },
};


